lang/java/reef-tests/src/test/java/org/apache/reef/tests/applications/vortex/addition/AdditionTestStart.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.reef.tests.applications.vortex.addition;
+
+import org.apache.reef.vortex.api.*;
+import org.apache.reef.vortex.examples.addition.AdditionAggregateFunction;
+import org.apache.reef.vortex.examples.addition.IdentityFunction;
+
+import javax.inject.Inject;
+import java.util.Vector;
+
+/**
+ * Test correctness of a addition aggregation test on Vortex.
```

Comment:
nit: the article `a` is misused. Actually, I think we can rephrase this to something like `Test correctness of an aggregation function that adds integer outputs on Vortex`. How does it sound?